Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -142,8 +168,9 @@ private static IEnumerable<Type> GetModelTypes(bool includeRequestModels)
 
                 if (includeRequestModels)
                 {
-                    var requestModels = methods.SelectMany(method => method.GetParameters(),
-                        (method, parameter) => parameter.ParameterType);
+                    var requestModels = methods
+                        .SelectMany(method => method.GetParameters(), (method, parameter) => parameter.ParameterType)
+                        .Where(t => t != typeof(Release)); // Release is a response model used as a parameter in IReleasesClient.UploadAsset
```

Comment:
Does this clause about `Release` type need to be here?  Eventhough it may be a Response model, if it is part of a client method parameters then it technically is a Request parameter as well, so probably shouldn't be excluded from Request model tests...